CHANGELOG.md

```diff
@@ -28,11 +28,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Changed repo config to `pyproject.toml`.
 * Changed `RhinoBrep.trimmed` to return single result or raise `BrepTrimmingError` instead of returning a list.
 * Changed order of imports according to `isort` and changed line length to `179`.
+* Changed use of `compas.geometry.allclose` to `compas.tolerance.TOL.is_allclose`.
+* Changed use of `compas.geometry.close` to `compas.tolerance.TOL.is_close`.
+* Changed imports of itertools to `compas.itertools` instead of `compas.utilities`.
 
 ### Removed
 
 * Removed `compas.scene.SceneObjectNode`, functionalities merged into `compas.scene.SceneObject`.
 * Removed `compas.scene.SceneTree`, functionalities merged into `compas.scene.Scene`.
+* Removed `compas.utilities.maps` since functionality is replaced by `compas.tolerance`.
```

Comment:
Could we have a more detailed entry here? In particular, I would highlight that the `geometric_key` function is gone and indicate what's the replacement